homeassistant/components/owntracks/device_tracker.py

```diff
@@ -147,5 +147,4 @@ async def async_added_to_hass(self):
     def update_data(self, data):
         """Mark the device as seen."""
         self._data = data
-        if self.hass:
```

Comment:
We can't remove this. It guards the entity interface which isn't ready until the entity has been added to home assistant.